ms/MSSel/test/tMSSelection.run

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+# -----------------------------------------------------------------------------
+# Usage: tMSSelection.run
+# -----------------------------------------------------------------------------
+# This script executes the program tMSSelection to test the
+# MSSelection module.
+#
+# $Id$
+#-----------------------------------------------------------------------------
+
+  #CASADEMO=`echo $CASAPATH | awk '{printf("%s/casacore.git/ms/MSSel/test/",$1)}'`+'/'
+  MS=$CASADEMO"mssel_test_small.ms"
+
+  BINPATH='/home/tara2/sanjay/casa_trunk_pristine/trunk/casacore.git/build/ms/MSSel/test/'
+
+  if [ ${#CASAPATH} = 0 ]
+  then
+     echo "UNTESTED: tMSSelection.run (CASAPATH not defined)"
+     exit 3
+  fi
+
+  function runCmd(){
+      #echo "Test:$1 "$2; 
+      eval "$2"
+      }
+
+  function getTestMS() {
+      tar zxf $testsrcdir/$MS.tgz
```

Comment:
OK. Can you list the issues that need attention?  The CASA specific
CASAPATH check should of course be removed.
On Sep 28, 2015 3:41 AM, "Julian Taylor" <notifications@github.com> wrote:

> In ms/MSSel/test/tMSSelection.run
> <https://github.com/casacore/casacore/pull/236#discussion_r40533977>:
>
> > +
> > +  BINPATH='/home/tara2/sanjay/casa_trunk_pristine/trunk/casacore.git/build/ms/MSSel/test/'
> > +
> > +  if [ ${#CASAPATH} = 0 ]
> > +  then
> > +     echo "UNTESTED: tMSSelection.run (CASAPATH not defined)"
> > +     exit 3
> > +  fi
> > +
> > +  function runCmd(){
> > +      #echo "Test:$1 "$2;
> > +      eval "$2"
> > +      }
> > +
> > +  function getTestMS() {
> > +      tar zxf $testsrcdir/$MS.tgz
>
> actually this should be fine, it were the other issues that threw me off
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/casacore/casacore/pull/236/files#r40533977>.[image:
> Web Bug from
> https://github.com/notifications/beacon/AM3Q7ZyWUohtTr_1Dyl0XvK-GxmdnREhks5o2QLjgaJpZM4F-kwZ.gif]
>